### PR TITLE
4434 – Update seeds script: create imported fact checks

### PR DESCRIPTION
**Pull request by vasconsaurus**
## Description

Add imported fact checks.

References: 4334

## What it looks like

Screenshot 2024-06-07 at 09 27 36
Screenshot 2024-06-07 at 09 27 48
Screenshot 2024-06-07 at 09 28 07

## How has this been tested?

By creating users from scratch and adding to an existing user.

## Things to pay attention to during code review

Please describe parts of the change that require extra attention during code review, for example:

- File FFFF, line LL: This refactoring does this and this. Is it consistent with how it’s implemented elsewhere?
- Etc.

## Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have added unit and feature tests, if the PR implements a new feature or otherwise would benefit from additional testing
- [ ] I have added regression tests, if the PR fixes a bug
- [ ] I have added logging, exception reporting, and custom tracing with any additional information required for debugging
- [ ] I considered secure coding practices when writing this code. Any security concerns are noted above.
- [ ] I have commented my code in hard-to-understand areas, if any
- [ ] I have made needed changes to the README
- [ ] My changes generate no new warnings
- [ ] If I added a third party module, I included a rationale for doing so and followed our current [guidelines](https://meedan.atlassian.net/wiki/spaces/ENG/overview#Choose-the-%E2%80%9Cright%E2%80%9D-3rd-party-module)

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -670,6 +685,21 @@ def cluster(project_media, feed, team_id)
...
+        summary: Faker::Company.catch_phrase,
+        title: Faker::Company.name,
+        user: bot,
+        language: 'en',
+        url: get_url_for_some_fact_checks(4)
+      }
+  end
+
+  def channel(media_type)
+    media_type == "Blank" ? { main: CheckChannels::ChannelCodes::FETCH } : { main: 0}
```
Comment (caiosba) [inline db/seeds.rb:701]: Would be nicer to replace `0` by `CheckChannels::ChannelCodes::MANUAL`, for readability.

**Review by caiosba — APPROVED** [review]
Left a suggestion, but not a blocker.

File: db/seeds.rb (hunk 1 above)
Comment (vasconsaurus) [reply db/seeds.rb:701]: Agreed, will update it.

**Comment by vasconsaurus** [comment]
I added the suggestion and fixed a small bug that was causing bot to be added as user to all the fact checks.

**Pull request merged**